Gem/Code/Source/Spawners/RoundRobinSpawner.cpp

```diff
@@ -30,6 +30,12 @@ namespace MultiplayerSample
             return AZStd::make_pair<Multiplayer::PrefabEntityId, AZ::Transform>(Multiplayer::PrefabEntityId(), AZ::Transform::CreateIdentity());
         }
 
+        if (m_spawnIndex >= m_spawners.size())
```

Comment:
For example the gamejam has this version of the code: https://github.com/aws-lumberyard-dev/o3de-multiplayersample/blob/GameJam_2022_07/Gem/Code/Source/Spawners/RoundRobinSpawner.cpp#L27

But this seems to be the o3de/o3de-multiplayerspawner version: https://github.com/o3de/o3de-multiplayersample/blob/development/Gem/Code/Source/Spawners/RoundRobinSpawner.cpp#L33